molecule/default/molecule.yml

```diff
@@ -0,0 +1,41 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: instance
+    image: centos:7
+    command: /sbin/init
+    tmpfs:
+      - /run
+      - /tmp
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+provisioner:
+  name: ansible
+verifier:
+  name: ansible
+lint: |
+  set -e
+  echo '==> yamllint'
+  yamllint .
+  echo '==> ansible-lint'
+  ansible-lint
+scenario:
+  test_sequence:
+    - dependency
+    - lint
+    - cleanup
+    - destroy
+    - syntax
+    - create
+    - prepare
+    - check
+    - converge
+    - idempotence
+    - check
+    - side_effect
+    - verify
+    - cleanup
+    - destroy
```

Comment:
I don't believe this section is required in a molecule.yaml file unless you have a need to manipulate the test sequence - which typically isn't something people do. Probably fine to remove this.